scala/opentransit/src/main/scala/com/azavea/opentransit/indicators/travelshed/JobsTravelshedIndicator.scala

```diff
@@ -245,13 +288,23 @@ object JobsTravelshedIndicator {
 
     // Reproject
     println(s"Reprojecting extent ${rasterExtent.extent} to WebMercator.")
-    val (rTile, rExtent) =
-      tile.reproject(rasterExtent.extent, crs, WebMercator)
-
-    println(s"Setting result of job indicator calculation to raster-cache-key $cacheId")
-    rasterCache.set(RasterCacheKey(JobsTravelshedIndicator.name + cacheId), (rTile, rExtent))
-
-    val regionalJobAccess = totalJobAccessResult / tileCount
-    regionalJobAccess
+    val (rBasicTile, rBasicExtent) =
+      basicTile.reproject(rasterExtent.extent, crs, WebMercator)
+    val (rAbsoluteTile, rAbsoluteExtent) =
+      absoluteTile.reproject(rasterExtent.extent, crs, WebMercator)
+    val (rPercentageTile, rPercentageExtent) =
+      percentageTile.reproject(rasterExtent.extent, crs, WebMercator)
+
+    println(s"Setting results of job indicator calculation to raster-cache-key $cacheId")
+    rasterCache.set(RasterCacheKey(name + cacheId), (rBasicTile, rBasicExtent))
+    rasterCache.set(RasterCacheKey(absoluteName + cacheId), (rAbsoluteTile, rAbsoluteExtent))
+    rasterCache.set(RasterCacheKey(percentageName + cacheId), (rPercentageTile, rPercentageExtent))
+
+    // return anonymous structure with named members
+    new {
+      val basic = basicTotalJobAccessResult / tileCount
+      val absolute = absolutePercentageTotalJobAccessResult / tileCount // average
+      val percentage = absolutePercentageTotalJobAccessResult / totalJobs
+    }
```

Comment:
I don't think there's anything wrong with this way of solving things (in fact, this would be preferable in clojure or js), but I think the scala community tends to prefer heavy utilization of case classes.

Perhaps at the top of this file, there could be something like:
```scala
case class JobAccessStatistics(basic: Double, absolute: Double, proportion: Double)
```
Then, `calculate` returns `JobAccessStatistics`.